tools/grammars/compiler/loader.go

```diff
@@ -122,7 +131,7 @@ func isValidGrammar(path string, info os.FileInfo) bool {
 	case ".tmlanguage", ".yaml-tmlanguage":
 		return true
 	case ".cson", ".json":
-		return strings.HasSuffix(dir, "/grammars") || strings.HasSuffix(dir, "/syntaxes")
+		return strings.HasSuffix(dir, "/grammars") || strings.HasSuffix(dir, "/syntaxes") || hasStringInArray(dir, GrammarsInNonStdPath)
```

Comment:
One could avoid the loop by defining `GrammarsInNonStdPath` and `IgnoredFiles` as `map[string]bool`.

Reply:
I thought about that, but changed my mind as it would mean using the full path to each grammar in the lists in `data.go` but now I've got fresh eyes on this, I know how I can do this.

Thanks for the suggestion. 🙇